cmd/swarm/main.go

```diff
@@ -376,6 +385,15 @@ pv(1) tool to get a progress bar:
 		// See config.go
 		DumpConfigCommand,
 	}
+
+	// append a hidden help subcommand to all commands that have subcommands
+	// if a help command was already defined above, that one will take precedence.
+	for i := range app.Commands {
+		cmd := &app.Commands[i]
+		if cmd.Subcommands != nil {
```

Comment:
I turned this into a recursive function.
However, this does not add anything to `swarm manifest add` since `swarm manifest add` does not actually have subcommands.

The CLI library ignores subcommands that have the same name that are added later, so we can save the check. I was trying to avoid a search for something that most probably won't be there.

Reply:
👍